### PR TITLE
docs: added some undocumented commands

### PR DESCRIPTION
**Pull request by KaiserKarel**
fixed some spelling and added some undocumented commands to the docs list.

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] I have read the [WRITING DOCUMENTATION](https://docs.ark.io/guidebook/contribution-guidelines/writing-documentation.html) guidelines
- [x] I have added necessary documentation

### CODE REVIEW

**Comment by ghost** [comment]
@KaiserKarel Thanks for submitting this pull request, a maintainer will get back to you shortly!

**Comment by ghost** [comment]
@adrian69 @faustbrian @supaiku0 - please review this in the next few days. Be sure to explicitly select labels so I know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

File: docs/guidebook/core/cli.md [hunk 1]
```diff
@@ -527,6 +543,12 @@ Start the forger
 ark forger:start
 ```
 
+or the equivalent, without invoking pm2:
+
+```bash
+ark forger:run
```
Comment (faustbrian) [inline docs/guidebook/core/cli.md:571]: The `--no-daemon` flag from `ark forger:start` is passed to `pm2` which does not result in the same as `ark forger:run`.

File: docs/guidebook/core/cli.md [hunk 2]
```diff
@@ -648,6 +693,12 @@ Start the relay
 ark relay:start
 ```
 
+or the equivalent without using pm2:
+
+```bash
+ark relay:run
```
Comment (faustbrian) [inline docs/guidebook/core/cli.md:721]: The `--no-daemon` flag from `ark relay:start` is passed to `pm2` which does not result in the same as `ark relay:run`.

**Review by faustbrian — CHANGES_REQUESTED** [review]

File: docs/guidebook/core/cli.md [hunk 3]
```diff
@@ -336,6 +346,12 @@ ark core:start --launchMode=seed
 ark core:start --no-daemon
 ```
 
+or use the following command, which supports the same set of flags.:
+
+```bash
+ark core:run
```
Comment (faustbrian) [inline docs/guidebook/core/cli.md:374]: The `--no-daemon` flag from `ark core:start` is passed to `pm2` which does not result in the same as `ark core:run`.

**Comment by ghost** [comment]
@KaiserKarel Your pull request needs some changes. Please wait for a comment from one of our developers for more information.

**Review by kristjank — CHANGES_REQUESTED** [review]
Can you add some more description / sentences to the available commands. So its more ELI5 friendly for the newcomers // 

Also now we can add/update tutorial "How to install ark-node - ark-core" (remove commander, and add cli docs.

**Comment by ghost** [comment]
Your pull request has been merged but was not assigned a bounty tier. A maintainer will assign a bounty tier to this pull request in the next few days.
